### PR TITLE
Lint the code for TRSP GraphDefinition.h based on the C and C++ google guidelines. 

### PR DESCRIPTION
**Pull request by XJTUmg**
src/trsp/src/GraphDefinition.h

### CODE REVIEW

File: src/trsp/src/GraphDefinition.h [hunk 1]
```diff
@@ -1,17 +1,20 @@
-#ifndef GRAPHDEFINITION_H
+#ifndef SRC_TRSP_SRC_GRAPHDEFINITION_H
 #define GRAPHDEFINITION_H
```
Comment (cvvergara) [inline src/trsp/src/GraphDefinition.h]: line 2 should be
#define SRC_TRSP_SRC_GRAPHDEFINITION_H

Comment (XJTUmg) [inline src/trsp/src/GraphDefinition.h]: Thanks.

**Comment by cvvergara** [comment]
Dont forget anbout the `#define SRC_TRSP_SRC_GRAPHDEFINITION_H`

File: src/trsp/src/GraphDefinition.h [hunk 2]
```diff
@@ -37,36 +40,35 @@ typedef struct path_element
...
     long ed_ind[2];
     long v_pos[2];
 } PARENT_PATH;
 
-typedef struct Rule{
+typedef struct Rule {
     double cost;
     std::vector<long> precedencelist;
     Rule(double c, std::vector<long> p) : cost(c), precedencelist(p) { }
 }Rule;
```
Comment (XJTUmg) [inline src/trsp/src/GraphDefinition.h]: No, it didn't.

**Pull request merged**